### PR TITLE
fix: 修复理智药插件中变量名错误；调整文档中理智药相关说明

### PR DESCRIPTION
**Pull request by status102**
- link to #7022

### CODE REVIEW

**Review by zzyyyl — APPROVED** [review]

**Comment by Constrat** [comment]
@status102 
```markdown
  - `Use Sanity Potion` specifies the number of sanity potions to use at most. Multiple medicines may be used at once.
  - `Use Originium` specifies the number of Originium to use at most. It is used one at a time.
```
Aren't potions also used one at a time?

File: docs/en-us/1.1-USER_MANUAL.md [hunk 1]
```diff
@@ -17,8 +17,8 @@
     <!--For the re-engraved SS event, you can enter `SSReopen-XX` to clear XX-1 ~ XX-9 levels at one time. Such as `SSReopen-IC`.-->
 
 - Fight options include `Use Sanity Potion`, `Use Originium`, `Perform Battles` and `Material`, you can specific any of them. The fight tasks stops once one of the specifications is met.
-  - `Use Sanity Potion` specifies the number of sanity potions to use at most. Note that the potion is usually used one at a time which is the one default by the game client when current sanity is not enough for the next fight. However, Santity Samplers offer a smaller amount of sanity and thus might be used more than one at a time. Pay attention to this when specifying the number of potions to use.
-  - `Use Originium` specifies the number of Originium to use at most. It is used one at a time.
+  - `Use Sanity Potion` specifies the number of sanity potions to use at most. Multiple medicines may be used at one time.
+  - `Use Originium` specifies the number of Originium to use at most. It is used one at one time.
```
Comment (Constrat) [inline docs/en-us/1.1-USER_MANUAL.md:21]:
```suggestion
  - `Use Sanity Potion` specifies the number of sanity potions to use at most. Multiple medicines may be used at a time.
  - `Use Originium` specifies the number of Originium to use at most. It is used one at a time.
```

**Comment by status102** [comment]
> @status102 
> ```markdown
>   - `Use Sanity Potion` specifies the number of sanity potions to use at most. Multiple medicines may be used at once.
>   - `Use Originium` specifies the number of Originium to use at most. It is used one at a time.
> ```
> Aren't potions also used one at a time?
> 

No, it was changed in #7022

File: docs/en-us/1.1-USER_MANUAL.md (hunk 1 above)
Comment (Constrat) [reply docs/en-us/1.1-USER_MANUAL.md:21]: Question: Aren't potions also used one at a time?

**Comment by Constrat** [comment]
So if I choose to use 2 sanity potions it will use them instantly together? Without waiting for the sanity cap?

File: docs/en-us/1.1-USER_MANUAL.md (hunk 1 above)
Comment (status102) [reply docs/en-us/1.1-USER_MANUAL.md:21]: Small potion may not have enough santy for stages. We used `using times` to limit actions. Now we need to set the count of potion.

**Comment by status102** [comment]
Oh, i forget to check strings in Gui.

**Review by Constrat — APPROVED** [review]

**Comment by Constrat** [comment]
> > So if I choose to use 2 sanity potions it will use them instantly together? Without waiting for the sanity cap?
> 
> won't check except DrGranxxxx mode ( forget the English name of `博朗台`

> should be DrGrandet 

Oh interesting

**Pull request merged**